smt/applications/podi.py

```diff
@@ -0,0 +1,475 @@
+
+    def get_left_basis(self) -> np.ndarray:
+        """
+        Getter for the left basis of the POD.
+
+        Returns
+        -------
+        left_basis : np.ndarray
+            Left basis of the POD.
+        """
+        return self.left_basis
+
+    def get_singular_values(self) -> np.ndarray:
+        """
+        Getter for the singular values from the Sigma matrix of the POD.
+
+        Returns
+        -------
+        singular_values : np.ndarray
+            Singular values of the POD.
+        """
+        return self.singular_values
+
+    def get_ev_ratio(self) -> float:
+        """
+        Getter for the explained variance ratio with the kept modes.
+
+        Returns
+        -------
+        EV_ratio : float
+            Explained variance ratio with the current kept modes.
+        """
+        return self.EV_ratio
+
+    def get_n_modes(self) -> int:
+        """
+        Getter for the number of modes kept during the POD.
+
+        Returns
+        -------
+        n_modes : int
+            number of modes kept during the POD.
+        """
+        return self.n_modes
+
+    def get_svd(self):
+        """
+        Getter for the svd object.
+
+        Returns
+        -------
+        svd
+        """
+        return self.svd
+
+    def set_interp_options(
+        self, interp_type: str = "KRG", interp_options_list: list = [{}]
+    ) -> None:
+        """
+        Set the options for the interpolation surrogate models used.
+
+        Parameters
+        ----------
+        interp_type : str
+            Name of the type of surrogate model that will be used for the whole set.
+            By default, the Kriging model is used (KRG).
+
+        interp_options_list : list[dict]
+            List containing dictionnaries for the options.
+            The k-th dictionnary corresponds to the options of the k-th interpolation model.
+            If the options are common to all surogate models, only a single dictionnary is required in the list.
+            The available options can be found in the documentation of the corresponding surrogate models.
+            By default, the print_global options are set to 'False'.
+
+        Example
+        --------
+        >>> interp_type = "KRG"
+        >>> dict1 = {'corr' : 'matern52', 'theta0' : [1e-2]}
+        >>> dict2 = {'poly' : 'quadratic'}
+        >>> interp_options_list = [dict1, dict2]
+        >>> sm.set_interp_options(interp_type, interp_options_list)
+        """
+
+        if not self.pod_computed:
+            raise RuntimeError(
+                "'compute_pod' method must have been succesfully executed before trying to set the models options."
+            )
+
+        if interp_type not in PODI_available_models.keys():
+            raise ValueError(
+                f"the surrogate model type should be one of the following : {', '.join(self.available_models_name)}"
+            )
+
+        if interp_options_list is None:
```

Comment:
We could use kriging as a default and do not require to use `set_inter_options` to set `interp_type` and/or `interp_options`.